### DISCUSSION TITLE
%E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1131020%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1

### DISCUSSION THREAD

**Discussion by giscus** [cit-hall-43/cit-hall-43.github.io#9] [Q&A]
# %E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1131020%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1

讚美主

https://cit-hall-43.github.io/%E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1131020%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1.html

**Comment by cit-hall-43** [comment]
第一首換補充本116
